### ISSUE TITLE
"Tutorial on fine-tuning using a pre-trained model" for  nnunetv2

### ISSUE THREAD

**Issue by kengoto1112** [MIC-DKFZ/nnUNet#1772]
Hi @FabianIsensee. 

I have read through #1108 tutorial, and it seems that these advices suggest that nnunetv2 may not be supported, as all the code starts with "nnUNet_plan_and_preprocess" or "nnUNet_train". 
Could you please provide guidance on how to adapt it for nnunetv2? 
Specifically, in nnUNetv2_plan_and_preprocess, options like -overwrite_plans_identifier are not available, which is causing some difficulty. 
Thank you for your assistance.

_Originally posted by @kengoto1112 in https://github.com/MIC-DKFZ/nnUNet/issues/1108#issuecomment-1784542166_

**Comment by constantinulrich** [comment]
Yes, this issue was opened last year before V2. But, lucky for you, in V2 it is way better documented: [Pretraining with nnU-Net](https://github.com/MIC-DKFZ/nnUNet/blob/master/documentation/pretraining_and_finetuning.md)

Does this help?
